1907-10-18.xml

```diff
@@ -0,0 +1,7379 @@
+                        <cell>about <date when="1907-11-28">28 Nov.</date></cell>
+                        
+                    </row>
+                    <row>
+                        <cell><name>Scharnhorst</name></cell>
+                        <cell><measure quantity="8200" unit="ton">8200</measure> Tons</cell>
+                        <cell>about <date when="1907-11-05">5 Nov.</date></cell>
+                        <cell><name>Bülow</name></cell>
+                        <cell><measure quantity="9000" unit="ton">9000</measure> Tons</cell>
+                        <cell>about <date when="1907-12-03">3 Dec.</date></cell>                     
+                    </row>
+                    <row role="label">
+                        <cell cols="3">OUTWARD: for CHINA and JAPAN via SUEZ, ADEN, COLOMBO, PENANG,
+                            SINGAPORE.</cell>
+                    </row>
+                    <row>
+                        <cell><name>Pr. Regt. Luitpold</name></cell>
+                        <cell><measure unit="ton" quantity="6300">6300</measure> Tons</cell>
+                        <cell>about <date when="1907-10-28">28 Oct.</date></cell>
+                    </row>
+                    <row>
+                        <cell><name>Yark</name></cell>
+                        <cell><measure unit="ton" quantity="9000">9000</measure> Tons</cell>
+                        <cell>about <date when="1907-11-11">11 Nov.</date></cell>
+                    </row>
+                    <row role="label">
+                        <cell cols="3">OUTWARD: for AUSTRALIA via SUEZ, ADEN, COLOMBO</cell>
+                    </row>
+                    <row>
+                        <cell><name>Bromon</name></cell>
+                        <cell><measure unit="ton" quantity="11600">11600</measure> Tons</cell>
+                        <cell>about <date when="1907-10-18">18 Oct.</date></cell>
+                    </row>
+                    <row>
+                        <cell><name>Seydlits</name></cell>
+                        <cell><measure unit="ton" quantity="8900">8900</measure> Tons</cell>
+                        <cell>about <date when="1907-11-15">15 Nov.</date></cell>
+                    </row>
+                </table>
+                <p><hi rend="bold">Service Maritime Roumain</hi></p>
+                <p>Regular Service from Alexandria to Smyrna (Mytilene), Constantinople and Constanza by the twin-screw steamers
+                Dacia and Imperatul Trsian; departures every fortnight, according to quarantine regulations in Turkey.</p>
+                <p><hi rend="bold">For particulars apply to the Agens, OTTO STERZING, Opera Square, Cairo. C. H. SCHOELLER &amp; Co. Besostrio-st., Alenxadria,
+                Messrs. THOS. COOK &amp; Son (Egypt) Ltd., are authorized to sell tickets in Cairo and Alexandria.</hi></p>
+                <p><measure type="indexNo">39-19-96</measure></p>
+            </div>
+            <div t
```

Comment:
cb n="5/2"? What do you mean?

Reply:
Ah yes, the 5/2 is supposed to be there. It's a little confusing to explain so I included a screenshot of the top of the page. If you notice all of the columns are fit to include 6 however for the advertisement (C. BRANDAUER), it changes from cols n="6" to cols n="3" due to the width change. Now regarding the cb n="5/2" is because of where it's placed in the page. Where it is placed is in between both the second and third column for cols n="3". This might seem confusing to you, it's hard to explain it but with how large the width of the advertisement is in comparison to the other ones around it, it can be placed in column breaks that aren't integers so in this case it's the 2.5 column.
![Screenshot_2](https://user-images.githubusercontent.com/54728307/67623847-0e935f00-f7f8-11e9-8761-76cd0208f5bf.jpg)

Reply:
Here is a graphic of my explanation, at the top is column 2.5, which is where my advertisement lies for <cols n="3">
![Screenshot_3](https://user-images.githubusercontent.com/54728307/67624146-17396480-f7fb-11e9-9d47-53f158f05dd8.jpg)

Reply:
This should be `<cols n="6"/> <cb n="4"/> <colSpan n="2"/>`. More info here: https://dig-eg-gaz.github.io/how-to/digitization/polished-tei-instructions/#3-column-breaks

Reply:
Oh awesome, I'll change it up real quick